components/event-service/config/config.go

```diff
@@ -139,6 +142,7 @@ func Configure() (*EventConfig, error) {
 	log.Debugf("feed endpoint: %s", config.HandlerEndpoints.Feed)
 	log.Debugf("cfg ingest endpoint: %s", config.HandlerEndpoints.CfgIngest)
 	log.Debugf("authz endpoint: %s", config.HandlerEndpoints.Authz)
+	log.Debugf("event feed endpoint: %s", config.HandlerEndpoints.EventFeed)
```

Comment:
Why log this twice in the same function?

Reply:
I am not sure why :man_shrugging: I was just following along with the other endpoints. I am going to remove this bottom set of logs for the endpoints.